gradle-language-server/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
   implementation "org.eclipse.lsp4j:org.eclipse.lsp4j.jsonrpc:0.12.0"
   implementation "org.codehaus.groovy:groovy-eclipse-batch:3.0.8-01"
   implementation "com.google.code.gson:gson:2.8.7"
+  implementation "org.apache.bcel:bcel:6.5.0"
```

Comment:
if it's shipped together, you should update `thirdpartynotice`

Reply:
should we update the `thirdpartynotice` for other dependencies?

Reply:
Yes, we'd better do that.

Reply:
track in #976